lib/kino_flame/runner_cell.ex

```diff
@@ -79,14 +135,21 @@ defmodule KinoFLAME.RunnerCell do
 
   @impl true
   def to_attrs(ctx) do
```

Comment:
The attrs we compute are all serialized in the .livemd, so we should only slice those relevant for the selected backend.

See similar logic in the [Database connection cell](https://github.com/livebook-dev/kino_db/blob/bb6af5bc11fec657cc76c974e6684ae48ac314ae/lib/kino_db/connection_cell.ex#L121).